### ISSUE TITLE
[BUG] error importing postgresql sql script

### ISSUE THREAD

**Issue by carlosmascarenhas** [drawdb-io/drawdb#291]
**Describe the bug**
When I try to import an SQL script generated by drawDB itself that has an auto-increment field, the following error is returned:
![image](https://github.com/user-attachments/assets/9c5177b7-9b13-4242-a367-ae28b1b0edbc)

Apparently it has to do with the PR #255 

**To Reproduce**
try to import this sql script that was generated by drawDB itself:

```
CREATE TABLE "test" (
	"id" INTEGER NOT NULL UNIQUE GENERATED BY DEFAULT AS IDENTITY,
	PRIMARY KEY("id")
);
```

**Comment by 1ilit** [comment]
#255 deals with exports. The parser fails on 'generated'. I'll report it if I haven't already

**Comment by Jal-GG** [comment]
@1ilit hey can i work on this if this issue is still open?

**Comment by 1ilit** [comment]
@Jal-GG yes but check if it's already fixed, the issue is in node-sql-parser not in drawdb
